### PR TITLE
New version: tastyworks.tastyworks version 2.3.1

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111749&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: tastyworks.tastyworks
```
Comment (stephengillie) [inline manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.installer.yaml:4]: The ISV appears to be TastyTrade, so shouldn't this be `TastyTrade.TastyWorks`?

File: manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+PackageIdentifier: tastyworks.tastyworks
+PackageVersion: 2.3.1
+PackageLocale: en-US
+Publisher: tastyworks # This is intentionally wrong to allow for automatic redirection to tastyworks.tastytrade
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: tastyworks # This is intentionally wrong to allow for automatic redirection to tastyworks.tastytrade
```
Comment (stephengillie) [inline manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.locale.en-US.yaml:12]: Is this like a tombstone, to facilitate redirection?

Comment (Trenly) [reply manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.locale.en-US.yaml:12]: Yes; Its described in the FAQ

**Review by stephengillie — APPROVED** [review]

**Pull request merged**